src/rp2_common/pico_stdio_uart/stdio_uart.c

```diff
@@ -128,13 +128,13 @@ void stdio_uart_deinit_full(struct uart_inst *uart, int tx_pin, int rx_pin) {
     uart_instance = uart;
     stdio_set_driver_enabled(&stdio_uart, false);
     uart_deinit(uart_instance);
-#if PICO_RP2040
+#if HAS_PADS_BANK0_ISOLATION
+// Leave pads isolated
+    if (tx_pin >= 0) hw_set_bits(&pads_bank0_hw->io[tx_pin], PADS_BANK0_GPIO0_ISO_BITS);
+if (rx_pin >= 0) hw_set_bits(&pads_bank0_hw->io[rx_pin], PADS_BANK0_GPIO0_ISO_BITS);
```

Comment:
```suggestion
    if (rx_pin >= 0) hw_set_bits(&pads_bank0_hw->io[rx_pin], PADS_BANK0_GPIO0_ISO_BITS);
```
Looks like you accidentally lost an indent.